aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -396,7 +396,7 @@ module aptos_framework::account {
         cap_rotate_key: vector<u8>,
         cap_update_table: vector<u8>,
     ) acquires Account, OriginatingAddress {
-        let addr = signer::address_of(account);
+        let addr = permissioned_signer::address_of(account);
         assert!(exists_at(addr), error::not_found(EACCOUNT_DOES_NOT_EXIST));
         check_rotation_permission(account);
```

Comment:
maybe make check_rotation_permission return address, for cleaner code?

Reply:
I'm planning to do that refactor in a separate PR.